src/bindings/node/node.ml

```diff
@@ -86,9 +86,15 @@ module ChildProcess = struct
 
   val get_stdin : t -> Stream.t [@@js.get "stdin"]
 
+  val killed : t -> bool [@@js.get "killed"]
+
   val on : t -> string -> (int -> unit) -> unit [@@js.call]
 
-  let on_close t close = on t "close" close
+  let on_close t f = on t "close" f
+
+  let on_error t f = on t "error" f
+
+  val kill : t -> string -> bool [@@js.call]
```

Comment:
Large binding changes should also be separate.

Reply:
I reverted this, it's not needed anymore